README.md

```
@@ -1,2 +1,68 @@
+   ```
+   ./gradlew :matchsdk:assembleRelease
+   ```
+   The AAR is generated at `matchsdk/build/outputs/aar/`.
+
+## Documentation
+Android Match SDK follows an implementation based on the MOSIP Biometric SDK specification (`IBioApiV2`).
+
+## Contribution & Community
+We welcome contributions from everyone!
+
+Check [here](https://docs.mosip.io/1.2.0/community/code-contributions) to learn how you can contribute code to this application.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)